templates/pod/profile/card.acl

```diff
@@ -0,0 +1,21 @@
+# ACL resource for the WebID profile document
+@prefix acl: <http://www.w3.org/ns/auth/acl#>.
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
```

Comment:
```suggestion
prefix : <#>
prefix acl: <http://www.w3.org/ns/auth/acl#>
prefix foaf: <http://xmlns.com/foaf/0.1/>
```
Use Turtle 1.1 syntax?
It was aligned to align Turtle's syntax to that of SPARQL, it seems like a good idea to use it by default.

Also, we could use the `:` prefix by default to make documents more readable.

Reply:
(see following two comments)

Reply:
@matthieubosquet Prob (not for us) is that rdflib.js is not compatible yet, as far as I know.

Reply:
Oh wow, you're right. I tried to look at fixing it, but rdflib.js is pretty confusing.

Is there a specific reason for it not using the standard N3 Parser? https://github.com/rdfjs/n3.js#parsing

We don't need to capitalize `prefix`, keywords are matched in a case-insensitive manner in the SPARQL grammar: https://www.w3.org/TR/sparql11-query/#sparqlGrammar

Reply:
I know that it's not needed (I wrote that standards-compliant parser 😉) but that's how it's conventionally done and most easily recognized.

For now, I'd just follow Postel's Law; N3.js also writes `@prefix` for that reason.

Reply:
Haha, true ^^,

Well, I should probably retract my comment for now then.